src/routes/index.ts

```diff
@@ -293,6 +294,8 @@ router.get(config.CONFIRMATION_URL, authentication, navigation.hasBOsOrMOs, conf
 // Routes for UPDATE journey
 router.get(config.UPDATE_LANDING_URL, updateLanding.get);
 
+router.get(config.RESUME_UPDATE_SUBMISSION_URL, authentication, resumeUpdateSubmission.get);
```

Comment:
Can you check that the auth code page is gone through on resume? And that the auth code needs to be entered again for the oe number?